test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -1755,4 +1755,140 @@ describe('v2/ion/uiSchemaTransformer', function () {
     });
   });
 
+  it('converts identify remidiation response', () => {
+    const rawIdentifyResponse = identifyResponse;
+    const transformedResponse  = {
+      'remediations': [
+        {
+          'name': 'identify',
+          'href': 'http://localhost:3000/idp/idx/identify',
+          'method': 'post',
+          'value': [
+            {
+              'name': 'identifier',
+              'label': 'Username',
+              'method': 'post'
+            },
+            {
+              'name': 'rememberMe',
+              'label': 'Remember Me',
+              'type': 'boolean',
+              'method': 'post'
+            }
+          ]
+        },
+        {
+          'name': 'select-enroll-profile',
+          'href': 'http://localhost:3000/idp/idx/enroll',
+          'method': 'post',
+          'value': []
+        }
+      ],
+      'neededToProceed': identifyResponse.remediation.value,
+      'actions': {},
+      'context': identifyResponse.context,
```

Comment:
`context: _.omit(identifyResponse, 'remediation', `cancel`)`

Reply:
`identifyResponse` has a lot more fields, hence plucked the `context` out manually.